python/testing/TestPythonCamera.py

```diff
@@ -25,11 +25,6 @@
 camera.getViewUp(vector)
 vector = camera.getViewUp()
 
-matrix = f3d.matrix4_t(1, 0, 0, 0, 0, 1, 0, 0, 0, 0, -1, 0, 0.5, -0.5, 10, 1)
-camera.setViewMatrix(matrix)
-camera.getViewMatrix(matrix)
-matrix = camera.getViewMatrix()
-
```

Comment:
missing tests

Reply:
The API is tested in C++.  
We need to improve python testing, but let's do it later, a lot of other bindings are not covered.

Reply:
indeed, lets open an issue to track this